planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -491,9 +509,19 @@ void ObstacleCruisePlannerNode::onTrajectory(const Trajectory::ConstSharedPtr ms
   //    (1) with a proper label
   //    (2) in front of ego
   //    (3) not too far from trajectory
-  const auto target_obstacles = convertToObstacles(traj_points);
+  const auto target_obstacles = [&]() {
```

Comment:
better to rename to sorted_target_obstacles, so that other people will know this variable's order has an implicit meaning.

Reply:
I decided to refactor a little and sort the moving and stopped obstacles in the findYieldCruiseObstacles function instead. I think it makes the code easier to read and it makes more sense semantically. Please let me know if you agree.